### PR TITLE
Hero args decorator

### PR DESCRIPTION
**Pull request by connormullett**
Closes issue #77. 

No functional changes, only refactoring and DRY'ing up commands.

### CODE REVIEW

**Review by cschuijt — CHANGES_REQUESTED** [review]
I have fixed the places I've commented on for your convenience. Things that jump out to me:
- /alias and /counter error message when giving no arguments uses the /build template
- /build, /counter and /alias give a double error message when no arguments are given

Fix these and it's merge ready as far as I'm concerned :)

**Comment by cschuijt** [comment]
Closes #74 and #75

File: src/bot/decorators/require_hero_args_decorator.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    Validates that a hero name or hero alias was given as args,
+    and retrieves the hero from the database. Fails if no
+    hero was given or if a hero doesn't exist.
+    """
+
+    @wraps(func)
+    def inner(update, context):
+        if not context.args:
+            update.message.reply_markdown_v2(
+                constants.MISSING_ARGUMENT_MESSAGE % "/build <hero name or alias>"
```
[the remark this answers is not in the document]
Comment (connormullett) [reply src/bot/decorators/require_hero_args_decorator.py]: Good catch, I've made MISSING_ARGUMENT_MESSAGE to be more generic and references the help command.

**Review by cschuijt — APPROVED** [review]
Should be all good now!

**Pull request merged**